openedx/tests/xblock_integration/proposed_test_case.py

```diff
@@ -0,0 +1,67 @@
+'''
+This is a simple XBlock test case.
+'''
+
+from xblock_testcase import XBlockTestCase
+
+olx = [
+    """<vertical>
+          <done url_name="done0"/>
+          <done url_name="done1"/>
+    </vertical>"""
+]
+
+
+class TestDone(XBlockTestCase):
+    test_configuration = [
+        {
+            "olx": olx,
+            "xblocks": [  # Stopgap until we handle OLX
```

Comment:
I agree that would be better. I'm not sure how I'd do that where the platform codebase is.